multi-class-classification/README.md

```diff
@@ -0,0 +1,26 @@
+# Binary classification: Is this bottle of wine red or white?
+This is the root directory for an example project for the
+[MLflow Classification Recipe](https://mlflow.org/docs/latest/recipes.html#classification-recipe).
```

Comment:
[MLflow Classification Recipe] --> [MLflow Multi Class Classification Recipe]

What do you think?